crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -941,13 +945,24 @@ impl HolochainP2pActor {
         let (s, r) = tokio::sync::oneshot::channel();
         self.pending.lock().unwrap().register(msg_id, s);
 
+        let start = std::time::Instant::now();
```

Comment:
This is duplicating a lot of what the `timing_trace_out!` macro calls are doing in the individual request code. Is it the right time to try to normalize that, or just leave the multiple Instants for now?

Reply:
Anything I can keep out of macro code, I'd like to. This is able to exist in a common place and I'd quite like it to stay where it is. As for merging those things together, yes very possibly but I guess they are going two different things in that they report to different places. So we'd want to share the `Instant` somehow.

I'm up for figuring that out, but next week because much tiredness and well after the end of my day already :D

Reply:
It's fine to normalize it out of the macro rather than into it. Essentially all the tracing is doing is printing out at trace level, so we could just do that next to your metric and delete the macro entirely.

But doesn't even have to be done in this PR.